src/Support/ClassLoader.php

```diff
@@ -94,34 +108,63 @@ public function __construct(Filesystem $files, $basePath, $manifestPath)
      * Load the given class file.
      *
      * @param  string  $class
-     * @return void
+     * @return true|null
```

Comment:
`return class_alias()` returns true or false. Can we also indicate what the different return values mean?